README.md

```diff
@@ -111,4 +111,89 @@ set max electrical add.                   float
 suction gas EB101-BT17                    float
 temporary lux                             boolean
 time factor                               float
-```
\ No newline at end of file
+```
+
+## store_psql.py
+
+Stores metrics in a postgresql database.
+
+It only inserts those metrics whose name is found to be a column of the `measurement` table and prints a warning for any other metrics found in the data. For metrics with duplicate names (e.g. "blocked"), a mapping can be applied, see the script for more details.
+
+Usage:
+
+```bash
+usage: store_psql.py [-h] [--host HOST] [--port PORT] [--database DATABASE]
+
+options:
+  -h, --help           show this help message and exit
+  --host HOST          defaults to localhost
+  --port PORT          defaults to 5432
+  --database DATABASE  defaults to "myupway"
+```
+
+Example:
+
+```bash
+cat examples/metrics.json | PSQL_USERNAME=myupway PSQL_PASSWORD=myupway python3 store_psql.py --host 10.110.1.6
+```
+
+Table schema of the `measurement` table:
+
+```
```

Comment:
Could you add `sql` here for proper code formatting?

Reply:
Good point, should be in there now!